src/bson.rs

```diff
@@ -970,80 +977,6 @@ impl Timestamp {
     }
 }
 
-/// Struct representing a BSON datetime.
```

Comment:
This was moved to its own `datetime` module as it was starting to get pretty big.